.github/workflows/ci-e2e-playwright.yml

```diff
@@ -7,6 +7,9 @@ name: E2E CI Playwright
 
 on:
     pull_request:
+    push:
+        branches:
+            - master
```

Comment:
hard to see when it broke since these run multiple times per PR
let's run on master too, since we want these not to flake